### ISSUE TITLE
Full screen toggle named incorrectly

### ISSUE THREAD

**Issue by joedolson** [ableplayer/ableplayer#586]
Reported by @digitala11y at https://github.com/ableplayer/ableplayer-wordpress/issues/8

> Here is the demo video link,
> https://www.digitala11y.com/academy/able-player-video-on-wordpress-demo/
> The full screen button is read as button by screen reader, there seem to be no programmatic label for the element.

The reported problem is a lack of announced label on the button; the button also appears to be wrapped with an inaccurate `aria-label` with the text 'Preferences'.
